README.md

```diff
@@ -32,6 +32,24 @@ const notifier = require('@dialonce/boot')().notifier;
 
 *Please note that these instructions will print an error if the module has not been initialised before*
 
+## Logger error/warn forward
+If you use the logger with error/warn level, the message is automatically
+forwarded to the notifier. You still have the ability to disable it globally or
+locally.
+
+```js
+const logger = require('@dialonce/boot')({ NOTIFY: false }).logger; // to disable it globally
```

Comment:
no by default it's enabled for all log with error/warn level, then you can disable it globally for all log this way, or locally (only for the current log) in log meta data.